apps/bfd-server/bfd-server-war/pom.xml

```diff
@@ -273,6 +273,10 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
```

Comment:
I think we need `<version>${slf4j.version}</version>` here to match the other poms.

Reply:
Might be why snyk is complaining

Reply:
I think you're right. I was having a bit of trouble with some failures do to the new logging I added for that NumberFormatException. Will add that now!